pages/settings/logsAndActivity.vue

```diff
@@ -146,64 +157,10 @@ export default Vue.extend({
   },
   beforeMount() {
     this.setPageTitle({ title: this.$t('settingsSections.logs.name') });
-    this.getActivities();
-    this.getLogs();
   },
   methods: {
     ...mapActions('page', ['setPageTitle']),
     ...mapActions('snackbar', ['pushSnackbarMessage']),
-    async getActivities(): Promise<void> {
-      this.loadingActivityStatus.status = 'loading';
```

Comment:
Maybe you can removed references to this and its logs version in the template and data?